.gitignore

```diff
@@ -7,3 +7,9 @@ vendor
 ## IntelliJ
 .idea
 *.iml
+
+## Generated code
+pkg/apis/nats/v1alpha2/zz_generated.deepcopy.go
```

Comment:
I think ok to include the generated code no? Otherwise would have to generate for every build for example

Reply:
Done.